tests/repositories/user_test.py

```diff
@@ -0,0 +1,57 @@
+from typing import TYPE_CHECKING
+
+from grouper.entities.user import User, UserMetadata
+from tests.constants import SSH_KEY_1, SSH_KEY_2
+from tests.util import key_to_public_key
+
+if TYPE_CHECKING:
+    from tests.setup import SetupTest
+
+
+def test_sql_all_enabled_users(setup):
+    # type: (SetupTest) -> None
+    """Test the implementation that does not use the graph.
+
+    This is not normally exercised by the usecase test suite, since the usecase expects to use the
+    graph.  We're not currently using the pure SQL implementation, but expect to soon in the
```

Comment:
The idea was that the frontend would do everything purely through SQL to avoid the write inconsistency problem with the graph, but the more I think about this, the less likely I think it will be that the frontend will ever need this specific function, since the user list wants less information (in particular, doesn't need to expand metadata or public keys), and the user details page needs a lot more. I decided to keep this implementation since I wrote it and tested it, but on further thought I'm going to delete it and replace it with `NotImplementedError()`, since I don't think we'll ever need it.